locales/en.yaml

```diff
@@ -290,6 +290,8 @@ wizard_community_enter_solvers_channel: Now please indicate the id or name of th
 wizard_community_success: Congratulatins, you have created your community!
 wizard_community_wrong_permissions: You do not have administrator permits for this group or channel
 wizard_exit: You have exit wizard mode, you can now write commands
+wizard_add_fiat_buy: recieve
+wizard_add_fiat_sell: pay
```

Comment:
*send

Reply:
"Please indicate the amount of EUR that you want to pay" is more clear than "to send".

Reply:
We can ask to more people in a public forum what would be the best word for this